packages/web/src/scss/components/Navigation/_NavigationLink.scss

```diff
@@ -0,0 +1,52 @@
+@use '@tokens' as tokens;
+@use '../../tools/typography';
+@use 'theme';
+
+.NavigationLink {
+    @include typography.generate(theme.$link-typography);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: var(--#{tokens.$css-variable-prefix}header-height);
+    padding-inline: theme.$link-spacing;
+    color: theme.$link-color-state-default;
+    border-top: theme.$link-border-bottom-width theme.$link-border-bottom-style transparent;
+    border-bottom: theme.$link-border-bottom-width theme.$link-border-bottom-style
+        theme.$link-border-bottom-color-style-state-default;
+    background-color: theme.$link-background-color-state-default;
+
+    @media (hover: hover) {
+        &:hover {
+            text-decoration: none;
+            color: theme.$link-color-state-hover;
+            background-color: theme.$link-background-color-state-hover;
```

Comment:
This is a tricky situation when the link is used inside `Header`: the link background overlaps the `Header` border.

I think this can be efficiently solved with a pseudo element for the bottom stripe (where did I see this…? 🤔). That would also eliminate the border-top mystification. But I'm definitely leaving the solution up to your consideration.

![image](https://github.com/user-attachments/assets/89bdf8b6-304e-4280-91ef-7de953840265)
![image](https://github.com/user-attachments/assets/69e40c77-7759-4a1c-9d57-e483b27a370c)

Reply:
The header is now a content-box, so the border is on the outside. I kept the link stripe as a border because it is consistent with FIgma, however, I asked Jirka about some details, so we might change this in the future.